CMake/run_qe.cmake

```diff
@@ -7,6 +10,10 @@ FUNCTION( ADD_QE_TEST TESTNAME PROCS TEST_BINARY NPOOL WORKDIR TEST_INPUT)
     ENDIF()
     SET_TESTS_PROPERTIES( ${TESTNAME} PROPERTIES ENVIRONMENT OMP_NUM_THREADS=1 PROCESSORS ${PROCS} WORKING_DIRECTORY ${WORKDIR} )
     SET_PROPERTY( TEST ${TESTNAME} APPEND PROPERTY LABELS "Quantum ESPRESSO" )
+    
+    SET(TEST_LABELS "")
+    ADD_TEST_LABELS( ${TESTNAME} TEST_LABELS )
```

Comment:
See three lines above. For cases will labels already applied, what is the behavior?

Reply:
It uses append so new properties should be added.  In general, set_property + append is safe and set_tests_properties is not.  I had to change the unit tests to set_property + append so labels would not be overwritten.

Reply:
More generally, is the output variable TEST_LABELS used anywhere?  If not, it doesn't seem necessary?